### ISSUE TITLE
Backslash and escaped double quote in string literals

### ISSUE THREAD

**Issue by shamrin** [witheve/lueve#4]
I've tried to import my data set:

```
import everything
  [#session-connect]
  freeze
    [#product available: true, price: 100, description: "text here", …]
    [#product available: true, price: 200, description: "another here", …]
    …
```

But I've failed at strings containing `"` quotes. `\"` silently fails:

```
silently fails on escaped double quote
  text = "escaped double quote \" here"
  maintain
    [#div text]
```

Plain `\` also fails:

```
silently fails on backslash
  text = "backslash \ here"
  maintain
    [#div text]
```

I think you need to switch to `tokenizer.eve` :-)

**Comment by joshuafcole** [comment]
You should be good to go now, please let us know if you have any further issues!

**Comment by shamrin** [comment]
It still fails for me :-(

None of the following examples work correctly (latest master: dcd8e9ebcb9c1c8f6f18ed7f32c02241e0ce715d):

```
doesn't even show the editor on localhost:8080 (Unexpected token   in JSON)
  text = "hello \\ world"
  maintain
    [#div text]
```

```
also no editor (Unexpected token   at JSON + Cannot read property 'up' of undefined)
  text = "hello \ world"
  maintain
    [#div text]
```

```
this one shows the editor, but doesn't produce "hello world" (Unexpected token w at JSON)
  text = "hello \" world"
  maintain
    [#div text]
```

```
no errors, but produces wrong output: "hello n world"
  text = "hello \n world"
  maintain
    [#div text]
```

**Comment by joshuafcole** [comment]
I expect the first 3 are the result of @convolvatron's json implementation not performing escaping yet when it dumps json strings.  The last looks like a parser issue.

**Comment by ibdknox** [comment]
I think it's just all of the various ways we try to escape and unescape, unfortunately there are a lot of them :/ I'll take a look today or tomorrow and get it fixed up.

**Comment by shamrin** [comment]
I believe you've mentioned the wrong issue in commit cd6a13d. Escaping bugs are still there.

**Comment by ibdknox** [comment]
You are correct. That should've been #6, my bad.

**Comment by ibdknox** [comment]
Ok, this should be fixed for real this time. Or at least your examples now work :)

**Comment by shamrin** [comment]
Thank you very much! Indeed these examples are working fine. Unfortunately, I couldn't yet test the fix with my data, because of #10.
